tests/qa-integration/pmm_pgsm_integration_test.js

```diff
@@ -140,12 +140,13 @@ Scenario(
 );
 
 Scenario(
-  'PMM-T1253 Verify pg_stat_monitor.pgsm_normalized_query settings @not-ui-pipeline @pgsm-pmm-integration @nazarov',
+  'PMM-T1253 Verify pg_stat_monitor.pgsm_normalized_query settings @pgsm-pmm-integration @nazarov',
```

Comment:
you need to have this not-ui-pipeline otherwise it will be picked up by regular pmm2-ui-test job

Reply:
added tag